tests/test_protobuf/jsondeserializationtest.cpp

```diff
@@ -319,172 +319,172 @@ TEST_F(JsonDeserializationTest, ComplexTypeSerializeTest)
 TEST_F(JsonDeserializationTest, RepeatedIntMessageTest)
 {
     RepeatedIntMessage test;
-    test.deserialize(serializer.get(), "{\"testRepeatedInt\":[0,1,321,-65999,123245,-3,3]}");
+    test.deserialize(serializer.get(), "{\"testRepeatedIntData\":[0,1,321,-65999,123245,-3,3]}");
```

Comment:
as long as you change serializer tests, that causes the broken compatibility with the original protobuf.

Reply:
I want to help further but this time I don't understand what to do. If I revert this change, the test's going to fail. Please help me with more instructions.

Reply:
@semlanik hello? Please help - I need to know what you want changed.

Reply:
Let me check your commit I'll back with fix soon :)

Reply:
Thank you for looking into this. :) By the way, I got qtprotobuf working with my real project last night. I have some questions which I cannot tell for certain are bugs yet so what's the best way to send them to you?

Reply:
You always may submit an issue to the project, I don't mind the bugs, etc. at all. I will determine how to deal with such kinds of questions. Eventually, there is the "question" category for issues, and I'm trying to answer ASAP.